win32.js

```diff
@@ -44,8 +44,14 @@ module.exports = {
       if (opts.appCopyright) {
         rcOpts['version-string'].LegalCopyright = opts.appCopyright
       }
+      
+      // add "asInvoker" to the built manifest by default
+      rcOpts['requested-executation-level'] = 'asInvoker'
+      if (opts.win32exeLevel) {
```

Comment:
Potentially we would want to be able to Strong Name the assembly: https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/strong-name-signing.md
There are also a few settings, such as dpiAware, and supportedOS that we may want to insert into the manifest.